File: DartProgramming/bin/m004_operator/Operator02.dart
//Operator Increment
void main() {
  int x = 0;

  //Pre-Increment "++x"
  //Plus 1, then, print data
  print('Value x before: $x');
  print('Pre-increment : ${++x}');
  print('Value x after : $x');

  //Post-Increment ""
  //Print data, then, plus 1
  print('Value x before: $x');
  print('Post-increment : ${x++}');
  print('Value x after : $x');
}

File: C_Programming/m004_operator/Operator02.c
//
// Created by Kelvin HT on 1/19/22.
//

#include <stdio.h>

int main() {

    // Operator Increment (++) And Decrement (--)
    int a = 10, b = 4;
    printf("Before : a = %d >-< b = %d \n", a, b);
    printf("After  : a = %d >-< b = %d \n", a--, b++);

    printf("Before : a = %d >-< b = %d \n", a, b);
    printf("After  : a = %d >-< b = %d \n", --a, ++b);

    return 0;
}
